### DISCUSSION TITLE
Adding giscus Comments - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#13] [General]
# Adding giscus Comments - Alan Pope's blog

My blog at popey.com/blog has gone through a number of iterations since I started it back in the 1990’s. First it was created using Microsoft FrontPage, and hosted on some free web space at CiX, and has morphed into a self-hosted WordPress site, Nikola static site, and now a Hugo static site.
At various times I’ve had comment systems available underneath posts. Over the years I’ve used Microsoft FrontPage extensions (which, amusingly used to crash the entire Web Host at CiX back in the day), WordPress comments, and Disqus to facilitate visitor interaction.

https://popey.com/blog/2022/12/adding-giscus-comments/

**Comment by vescovaccio** [comment]
Very nice! 👏

**Comment by skellat** [comment]
This is something to perhaps implement on my sites, I think.

**Comment by TimTaylor** [comment]
This is very cool! Interested to see how it goes.

**Comment by filviu** [comment]
I'm curious what pushed you from Nikola to Hugo? I went wordpress -> hugo -> wordpress and now I'm considering going back to a static site and I'm curious what made you choose hugo among the myriad options?

**Reply by popey** [reply]
I initially liked Nikola, but found it a but clunky. Hugo is very simple and easy to use.
